axon-framework/modules.md

```diff
@@ -20,18 +20,14 @@ Axon 'Main Modules' are the modules that have been thoroughly tested and are rob
 >
 > During the migration of Java EE to the Eclipse Foundation, ["Eclipse and Oracle have agreed that the `javax` package namespace cannot be evolved by the Jakarta EE community"](https://eclipse-foundation.blog/2019/05/03/jakarta-ee-java-trademarks/). With the release of the Jakarta EE 9 Platform, the `javax` package names were officially and finally renamed as `jakarta`. The change impacts a few APIs that Axon Framework uses, like `inject-api`, `persistence-api`, `validation-api`. Furthermore, it impacts third-party technologies often used with Axon, such as Spring and Hibernate.
 >
->To give developers a choice, Axon Framework's core modules affected by the change now come in two variations. The ones with `-jakarta` suffix use the new `jakarta` packages. Those without the suffix depend on the old `javax` packages.
+> All code that uses the affected annotations has been updated to use the `jakarta` namespace. The variants using a `javax` namespace have been deprecated. They can be found in `legacy_jpa` packages across the Messaging, Modeling, Event Sourcing and Spring Boot Starter modules. Since we moved to using the `jakarta` namespace by default, the `-jakarta` modules introduced with 4.6.0 are no longer available. If you are using Spring auto configuration, it should still use the `javax` namespace, as long as you are using Spring Boot 2. This is done by checking the `EntityProvider` class provided by Spring Boot.
```

Comment:
I guess it should be `EntityManager` i.o. `EntityProvider`, right?
```suggestion
> All code that uses the affected annotations has been updated to use the `jakarta` namespace. The variants using a `javax` namespace have been deprecated. They can be found in `legacy_jpa` packages across the Messaging, Modeling, Event Sourcing and Spring Boot Starter modules. Since we moved to using the `jakarta` namespace by default, the `-jakarta` modules introduced with 4.6.0 are no longer available. If you are using Spring auto configuration, it should still use the `javax` namespace, as long as you are using Spring Boot 2. This is done by checking the `EntityManager` class provided by Spring Boot.
```

Reply:
actually it's the `EntityManagerFactory` class.